### DISCUSSION TITLE
Issue with CORS and 429 status

### DISCUSSION THREAD

**Discussion by mmoore99** [trakt/trakt-api#441] [Q&A] [answered]
@rectifyer I have an app that works fine with the Trakt API except with regard to 429 responses. When a 429 response is sent to the app a CORS error is generated that says "failed to load resource access control allow origin status code 429". Because of this the app does not see the 429 status code and does not get the rate limit response headers. Is there something that needs to be done on the Trakt server regarding this or is there something that I can do in the app?

**Comment by mmoore99** [comment]
@rectifyer can you shed some light on this?

**Comment by rectifyer** [comment]
I'm looking into this. The rate limiting is handled before it actually gets the app code, so looking to see what I can do.

**Reply by rectifyer** [reply, marked as answer]
The API has been updated to return the correct headers for a CORS call and should allow you to read the rate limit headers properly.

**Reply by mmoore99** [reply]
Yes, that solved the issue. Thanks for looking into this!
